src/assets/scripts/client/tutorial/TutorialView.js

```diff
@@ -403,6 +403,40 @@ export default class TutorialView {
         });
 
         this.tutorial_step({
+            title: 'Say Route',
+            text: ['With the capability to edit the route, you obviously will need a way to know what their current route is. Typically, this is displayed',
+                   'in the flight progress strip. However, to preserve screen space, you will instead have to check the route by this command &lsquo;route&rsquo;',
```

Comment:
this sentence:
>However, to preserve screen space, you will instead have to check the route by this command &lsquo;route&rsquo;'

can be improved to:
>However, to preserve screen space, you can have the aircraft tell you their route with the say route command  &lsquo;sr&rsquo;'

the command here should be `sr` which stands for `say route`